Hearthstone Deck Tracker/GameEventHandler.cs

```diff
@@ -1048,12 +1048,16 @@ public void HandleOpponentGetToDeck(Entity entity, int turn)
 
 		public void HandleOpponentRemoveFromPlay(Entity entity, int turn) => _game.Player.RemoveFromPlay(entity, turn);
 
-		public void HandlePlayerPlayToGraveyard(Entity entity, string cardId, int turn) => _game.Player.PlayToGraveyard(entity, cardId, turn);
+		public void HandlePlayerPlayToGraveyard(Entity entity, string cardId, int turn)
+		{
+			_game.Player.PlayToGraveyard(entity, cardId, turn);
+			GameEvents.OnPlayerPlayToGraveyard.Execute(entity.Card);
```

Comment:
Please use `(Card)entity.Card.Clone()` instead of passing the original card object here (same for opponent).